spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -13,11 +13,16 @@ module Graphql
     end
 
     before do
-      create(:freak, first_name: 'Ion')
-      create(:freak, first_name: 'Gheorghe')
+      create(:freak)
     end
 
-    it { is_expected.to match_response_for(query: :freaks, sample: :freaks) }
+    context 'with freak role' do
```

Comment:
Considering the context, I'd expect outside the `freak` outside to have no role and inside of it to have a `role`, but I don't see any role being created inside the context & assigned to the `freak`.

Either make that clear for the context or move the test outside the context.